vizro-ai/docs/index.md

```diff
@@ -1,47 +1,32 @@
 # Vizro-AI
 
-Vizro-AI is a tool designed for generating data visualizations. It serves as an extension to Vizro, leveraging natural language capabilities to empower users in creating charts effortlessly.
+Vizro-AI extends [Vizro](https://vizro.readthedocs.io) with a user-friendly interface that uses natural language capabilities to help you create interactive charts effortlessly.
```

Comment:
What do we consider as user-friendly interface here? Because Vizro-AI doesn't provide a user-friendly interface out of the box, does it? The user actually has to create it e.g. the chatbot was a use case we had to configure, but the vizro-ai package itself doesn't provide any UI interface. Or does this not refer to a UI interface?

Reply:
That's a good point! I've made an edit to revise this.